misc/swagger-ballerina/modules/swagger-to-ballerina-generator/src/main/java/org/ballerinalang/swagger/utils/CodegenUtils.java

```diff
@@ -88,4 +89,17 @@ public static void writeFile(Path filePath, String content) throws IOException {
             }
         }
     }
+    
+    /**
+     * Removes underscores and hyphens for identifiers.
+     * @param identifier Path
+     * @return Cleaned identifier.
+     */
+    public static String normalizeForBIdentifier(@Nullable String identifier){
+        if (identifier == null) {
```

Comment:
Whats the scenario(s) the identifier becomes null?

Reply:
This is used in setting the resource name either from swagger operationId or using the swagger path of the resource. If the operationId is assign null by this method, then the swagger path is used. This is the same implementation as done in https://github.com/ballerina-platform/ballerina-lang/pull/10455/files#diff-22c4dc9aa3a121e766d6cfb95aaa8070L143. I moved it to a common location.